chrome/content/zotero-better-bibtex/cayw.coffee

```diff
@@ -186,7 +186,7 @@ Zotero.BetterBibTeX.CAYW.Formatter = {
       cite += "#{citation.prefix} " if citation.prefix
       cite += '-' if citation['suppress-author']
       cite += "@#{citation.citekey}"
-      cite += ", #{citation.label} #{citation.locator}" if citation.locator
+      cite += " #{citation.locator}" if citation.locator
```

Comment:
http://pandoc.org/demo/example19/Extension-citations.html (see "Citations go inside square brackets and are separated by semicolons.") includes the comma, are you sure about this?